src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,47 @@
+import React, { Component } from 'react'
+
+import './Carousel.css'
+
+class Carousel extends Component {
+    state = {
+        position: 0
+    }
+
+    prev() {
+        if ((this.state.position - 1) < 0){
```

Comment:
este if está bien, pero si haces un ternario dentro del setState queda más guapo:

```suggestion
        this.setState({
            position: (this.state.position - 1) < 0 ?
            (images.length - 1)
            :
            (this.state.position - 1)
        })
```